src/env.js

```diff
@@ -26,6 +28,8 @@ export const env = createEnv({
   runtimeEnv: {
     NODE_ENV: process.env.NODE_ENV,
     // NEXT_PUBLIC_CLIENTVAR: process.env.NEXT_PUBLIC_CLIENTVAR,
+    NEXT_PUBLIC_POSTHOG_KEY: process.env.NEXT_PUBLIC_POSTHOG_KEY,
+    NEXT_PUBLIC_POSTHOG_HOST: process.env.NEXT_PUBLIC_POSTHOG_HOST,
```

Comment:
Does this need to be set to use the proxy?

Reply:
> Adds PostHog to the landing page. This uses a next.js rewrite to proxy the requests through and prevent blocking

I mean I guess it's not really a proxy. I'm just wondering if we need this env variable, or if we can just use the default host and simplify things.

Reply:
Oh this is to prevent adblockers from the analytics ingest. This actually *is* a proxy, and the rewrites route requests from Next to Posthog. The request flow is like so:

Posthog Client -> Netlify (at HOST/ingest endpoint etc) -> Posthog API